crates/ruff/src/rules/pyflakes/rules/f_string_missing_placeholders.rs

```diff
@@ -8,6 +8,26 @@ use crate::registry::Diagnostic;
 use crate::violation::AlwaysAutofixableViolation;
 
 define_violation!(
+    /// ## What it does
+    /// Checks for f-strings that do not contain any placeholder expressions.
+    ///
+    /// ## Why is this bad?
+    /// F-strings are a convenient way to format strings, but they are not
+    /// necessary if there are no placeholder expressions to format. In this case,
+    /// a regular string should be used instead.
```

Comment:
Again nit-picking but this doesn't say why using f-strings without placeholders is bad ... just the fact that you don't have to use something doesn't say that using it is bad.

Disadvantages I can think of: There's probably some runtime overhead since Python has to parse the string looking for placeholders, and it's a bit misleading/confusing for readers since you'd expect a placeholder.